### DISCUSSION TITLE
建议与需求

### DISCUSSION THREAD

**Discussion by QYG2297248353** [metatube-community/metatube-community.github.io#6] [General]
## Koyeb 快速部署

1. 增加一键部署按钮
2. 更新部署文档，页面变化较大

### 界面变化
![image](https://github.com/user-attachments/assets/9c36e7e6-e5ea-40e4-bd2f-8c55cf331343)

### 一键部署按钮
[![Deploy to Koyeb](https://www.koyeb.com/static/images/deploy/button.svg)](https://app.koyeb.com/deploy?name=metatube-server&type=docker&image=docker.io%2Fmetatube%2Fmetatube-server%3Alatest&instance_type=free&regions=was)

## 增加 API 文档
支持开发者，了解并使用 MetaTube API 调用服务
有文档说明，方便开发者调用自部署应用服务，文档不方便，如果可以接入类似 swagger 服务也可，或可利用 apidfox 发布接口文档

小小建议，望采纳

**Comment by xjasonlyu** [comment]
Koyeb 其实已经不适合部署了，所以更新这部分文档意义不大。

**Comment by QYG2297248353** [comment]
> Koyeb 其实已经不适合部署了，所以更新这部分文档意义不大。

那还有免费的云部署吗，本地部署，网络环境太麻烦了🙈

其他的都在收费，唉

**Comment by 9haoke** [comment]
> Koyeb 其实已经不适合部署了，所以更新这部分文档意义不大。

为什么koyeb已经不适合部署呢？

**Comment by QYG2297248353** [comment]
> > Koyeb 其实已经不适合部署了，所以更新这部分文档意义不大。
> 
> 为什么koyeb已经不适合部署呢？

我试过才知道，国内无法访问，要套一层 cf

**Comment by 9haoke** [comment]
> > > Koyeb 其实已经不适合部署了，所以更新这部分文档意义不大。
> > 
> > 
> > 为什么koyeb已经不适合部署呢？
> 
> 我试过才知道，国内无法访问，要套一层 cf

就是你在koyeb上部署了后端，然后在插件上填了后端地址，在不出国的网络环境下还是刮不了？
